client/src/HowItWorks.tsx

```diff
@@ -1,7 +1,341 @@
+                        onClick={() => handleClickTab("nonprofit")}
+                    >
+                        <h3 className={`${classes.button} ${tabSelected === "nonprofit" ? classes.selected : ''}`}
+                        >I'm on a nonprofit team</h3>
+                    </Grid>
+                    <Grid container item sm={6} xs={12}
+                        onClick={() => handleClickTab("citizen")}
+                    >
+                        <h3 className={`${classes.button} ${tabSelected === "citizen" ? classes.selected : ''}`}
+                        >I'm a citizen</h3>
+                    </Grid>
+                </Grid>
+            </Box>
+
+            <Box className={`${classes.nonprofitContainer} ${classes.mainPageSection} ${classes.limitWidth}`}>
+                <InstructionGrid instructionList={ tabSelected === "nonprofit" ? nonprofitInstructionList : citizenInstructionList }></InstructionGrid>
+            </Box>
+
+            <Box className={`${classes.greySection} ${classes.mainPageSection}`}>
+                <Box className={`${classes.limitWidth}`}>
+                    <Typography variant="h3" component="h3" align="left">
+                        { tabSelected === "nonprofit" ? nonprofitBulletList.title : citizenBulletList.title }
+                    </Typography>
+                    <BulletGrid list={tabSelected === "nonprofit"? nonprofitBulletList.list : citizenBulletList.list}></BulletGrid>
+                </Box>
+            </Box>
+
+            <Box className={`${classes.mainPageSection}`}>
+                <Box className={`${classes.limitWidth} ${classes.questionSection}`}>
+                    <Typography variant="h3" component="h3" align="left">
+                        Still have questions?
+                    </Typography>
+                    <Box className={classes.questions}>
+                        <QuestionList questionList={tabSelected === "nonprofit" ? nonprofitQuestions : citizenQuestions}></QuestionList>
+                    </Box>
+                </Box>
+            </Box>
+
+        </>
+    );
 }
 
+
+
+
+// SUB-COMPONENT InstructionGrid
+
+const instructionStyles = makeStyles<Theme, InstructionProps> ({
+    headerText: {
+        width: '100%',
+        maxWidth: '1100px',
+        fontSize: '1.3rem',
+        textAlign: 'left',
+    },
+    gridBox: {
+        width: '100%',
+        overflow: 'hidden',
+        padding: '20px',
+        '& img': {
+            maxWidth: '100%',
+            height: 'auto',
+        },
+        '& h3': {
+            paddingBottom: '1em',
+        },
+    },
+});
+
+type InstructionProps = {
+    instructionList: { title: string, body: string, image: string }[],
+};
+
+function InstructionGrid(props: InstructionProps) {
+    const classes = instructionStyles(props);
+    var row = 0;
+
+    return (
+        <Grid container justify='space-between'>
+            {props.instructionList.map((instructionItem) => {
```

Comment:
use the map's index (`.map(instructionItem, index) =>`) instead of `row`

Reply:
Thanks!